### PR TITLE
Prepare repository for VAST 2021.04.29-rc3

### PR DESCRIPTION
**Pull request by dominiklohmann**
###  :notebook_with_decorative_cover: Description

This PR was created automatically using the `./scripts/create-release 2021.04.29-rc3` from #1582.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

n/t

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -97,6 +104,21 @@ This changelog documents all notable changes to VAST and is updated on every rel
...
+- We fixed a regression that made it impossible to build static binaries from outside of the repository root directory.
+  [#1573](https://github.com/tenzir/vast/pull/1573)
+
+- The `VASTRegisterPlugin` CMake function now correctly removes the `ENTRYPOINT` from the given `SOURCES`, allowing for plugin developers to easily glob for sources again.
+  [#1573](https://github.com/tenzir/vast/pull/1573)
+
+- The `exporter.selectivity` metric is now 1.0 instead of NaN for idle periods.
+  [#1574](https://github.com/tenzir/vast/pull/1574)
+
+- VAST no longer renders JSON numbers with non-finite numbers as `NaN`, `-NaN`, `inf`, or `-inf`, resulting in invalid JSON output. Instead, such numbers are now rendered as `null`.
```
Comment (lava) [inline CHANGELOG.md:116]: This looks like a change, if not a breaking change. Not completely sure in which PR such changes should go.

**Review by lava — APPROVED** [review]
Looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (dominiklohmann) [reply CHANGELOG.md:116]: It was invalid JSON before, so this is most certainly a bug fix.

Comment (dominiklohmann) [reply CHANGELOG.md:116]: >  in which PR such category changes should go

In this one, modified by hand.

But as I said above, this is a bug fix as the output was invalid JSON before.

**Pull request merged**